src/lib/api.ts

```diff
@@ -83,14 +83,12 @@ async function handleResponse(res: Response) {
     browser &&
       sessionStorage.setItem('logout-reason', 'Your session has timed out. Please log in.');
     logout(undefined, true);
-    return;
```

Comment:
I don't think we want to throw the error on 401s. It causes the UI to get stuck in loops.

Reply:
We have a different story to address the logging out on 403, but the success message is just extra confusing.